modules/apps/josh-lapurd-theme/src/WEB-INF/src/resources-importer/journal/articles/img_Hero/img_Hero.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+
+<root available-locales="en_US" default-locale="en_US">
+	<dynamic-element name="img_Hero" type="image" index-type="keyword" instance-id="mkxl">
```

Comment:
Structure field id's should be all lower case since we have the underscore to separate the words.

Ex.
```
img_hero
```